scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
We need to fix that crowbarctl command for the upgrade case anyway, as we need asynchronous API calls.
I think --timeout 300 did not work for me and I do not see a point for researching good timeout value.